packages/flutter/lib/src/material/text_theme.dart

```diff
@@ -355,12 +355,20 @@ class TextTheme with Diagnosticable {
   /// [displaySmall], [headlineLarge], [headlineMedium], and [bodySmall]. The
   /// `bodyColor` is applied to the remaining text styles.
   ///
+  /// The `displayFontFamily` is applied to [displayLarge], [displayMedium],
+  /// [displaySmall], [headlineLarge], [headlineMedium], and [bodySmall]. The
+  /// `bodyFontFamily` is applied to the remaining text styles.
```

Comment:
What are the remaining style? And how do these relate to fontFamily and fontFamilyFallback?

Reply:
The other styles include
`headlineSmall`
`titleLarge`
`titleMedium`
`titleSmall`
`bodyLarge`
`bodyMedium`
`bodySmall`
`labelLarge`
`labelMedium`
`labelSmall`

This is the config that `displayColor` follows.